run.sh

```diff
@@ -184,6 +189,10 @@ if [ $PARALLEL == 1 ]; then
   fi
 else # Not using multiple CircleCI containers, just queue up the tests in sequence
   if [ "$CI" != "true" ] || [ $CIRCLE_NODE_INDEX == 0 ]; then
+    if [ $UPLOAD == 1 ]; then
+      # Clear out screenshots-i18n/ directory
+      rm -f screenshots-i18n/*
```

Comment:
I think it's probably fine to leave this hard-coded here for now since this is only used for the i18n screenshots, but what do you think about replacing this with the SCREENSHOTDIR envvar?